sentry.go

```diff
@@ -79,14 +85,29 @@ func (so *SentryOutput) getClient(dsn string) (client *raven.Client, err error)
 
 func (so *SentryOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper) (err error) {
 	var (
-		e      error
-		pack   *pipeline.PipelinePack
-		client *raven.Client
+		e        error
+		pack     *pipeline.PipelinePack
+		client   *raven.Client
+		contents []byte
 	)
 
+	encoder := or.Encoder()
+	if encoder == nil {
+		payloadEncoder := new(plugins.PayloadEncoder)
+		payloadEncoder.Init(new(plugins.PayloadEncoderConfig))
+		encoder = payloadEncoder
+	}
```

Comment:
Instantiating an encoder by hand isn't a great idea. The usual pattern is to simply return w/ an error if an encoder isn't specified.